docs/examples/advancedChoiceWorld.m

```diff
@@ -122,9 +122,13 @@ function advancedChoiceWorld(t, evts, p, vs, in, out, audio)
 evts.azimuth = azimuth;
 evts.response = response;
 evts.feedback = feedback;
+evts.reward = reward;
 evts.interactiveOn = interactiveOn;
 % Accumulate reward signals and append microlitre units
-evts.totalReward = out.reward.scan(@plus, 0).map(fun.partial(@sprintf, '%.1f�l')); 
+evts.totalReward = out.reward.scan(@plus, 0).map(@(x) sprintf('%.1ful', x));
+% the commented out line below would also work
+% evts.totalReward = out.reward.scan(@plus, 0).map(fun.partial(@sprintf, '%.1f�l'));
+evts.vs = vs;
```

Comment:
What's this for?  We don't usually save the visual object, perhaps it would be more efficient to simply save the phase as an event?

Reply:
hmm yeah I was looking at something with this, but forget exactly what. I'll remove it